pkg/vulnerabilityreport/builder_test.go

```diff
@@ -207,7 +209,9 @@ func TestScanJobBuilder(t *testing.T) {
 							trivyoperator.LabelResourceSpecHash:           "7d6c657bf7",
 						},
 					},
-					Spec: corev1.PodSpec{},
+					Spec: corev1.PodSpec{
+						SecurityContext: &corev1.PodSecurityContext{},
```

Comment:
I don't think this test should change, if I don't misunderstand the test. As this new feature should not change the existing job spec, if not using this feature. Multiple occurrences.

Reply:
ok you prefer to keep previous behaviour, and not add any empty spec, let's look at this